improver_tests/psychrometric_calculations/test_PhaseChangeLevel.py

```diff
@@ -415,6 +407,27 @@ def test_basic(self):
         result = plugin.find_max_in_nbhood_orography(self.cube)
         self.assertArrayAlmostEqual(result.data, self.expected_data)
 
+    def test_null(self):
+        """Test the function does nothing when radius is zero."""
+        plugin = PhaseChangeLevel(phase_change="snow-sleet", grid_point_radius=0)
+        expected_data = self.cube.data.copy()
+        result = plugin.find_max_in_nbhood_orography(self.cube)
+        self.assertArrayAlmostEqual(result.data, expected_data)
+
+    def test_null_lat_lon(self):
```

Comment:
Why would you expect this to behave differently to the above?

Reply:
I don't, but that wasn't the point. The reason for this PR is to enable some support for lat-lon domains and I wanted a test to show that it did indeed work for a lat-lon domain if the grid-radius were set to zero. To try to make this behaviour a bit clearer, I've added a fourth test showing the error you can expect if you try this.

Reply:
Ah OK.  In which case can I suggest a very small change to the docstring: `Test the function succeeds and does nothing when radius is zero and grid is lat-lon.`.

Reply:
Done